event-individual.html

```diff
@@ -0,0 +1,81 @@
+        </nav>
+
+        <nav title="Path">
+              <a title="Home Page" href="index.html">Acasa<img width="20" height="20" src="https://img.icons8.com/material-outlined/24/forward.png" alt="Forward Arrow"/></a>
+              <a title="All Events-Page" href="events.html">Evenimente<img width="20" height="20" src="https://img.icons8.com/material-outlined/24/forward.png" alt="Forward Arrow"/></a>
+              <a title="Event Page" href="#">Nume Eveniment</a>
+        </nav>
+
+        <section title="Title">Event Name</section>
+        <section title="Event Photo"><img src="https://media.sproutsocial.com/uploads/2017/02/10x-featured-social-media-image-size.png" alt="Event Photo"/></section>
+        <section title="Share">Share<img width="20" height="20" src="https://img.icons8.com/ios-glyphs/30/share--v1.png" alt="Share Icon"/></section>
+        <section title="Event Info">
+            <div title="Date"><img width="20" height="20" src="https://img.icons8.com/sf-regular/48/calendar.png" alt="Calendar Icon"/>Joi, 23 Martie</div>
+            <div title="Location">
+                <p title="City"><img width="20" height="20" src="https://img.icons8.com/ios-filled/50/marker.png" alt="Location Icon"/>Bucuresti</p>
+                <p title="Place">Parcul Natural Vacaresti</p>
+            </div>
+            <div title="Phone Number">0720 000 000</div>
+            <div title="Map"><img width="250" height="250" src="https://i0.wp.com/www.cssscript.com/wp-content/uploads/2018/03/Simple-Location-Picker.png?fit=561%2C421&ssl=1" a="Location Map"/></div>
+        </section>
+        <section title="content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut varius molestie ante, eu elementum sapien congue et. Suspendisse feugiat tempor sagittis. Morbi felis lorem, sagittis nec tellus sed, dignissim scelerisque turpis.
+
+            Pellentesque blandit dignissim cursus. Vestibulum nibh diam, venenatis vel eros in, egestas ullamcorper libero. Quisque id ante erat. Mauris fermentum ex vitae tristique volutpat.
+            
+            Suspendisse feugiat tempor sagittis. Morbi felis lorem, sagittis nec tellus sed, dignissim scelerisque turpis.
+            
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut varius molestie ante, eu elementum sapien congue et. Suspendisse feugiat tempor sagittis. Morbi felis lorem, sagittis nec tellus sed, dignissim scelerisque turpis.
+            
+            Pellentesque blandit dignissim cursus. Vestibulum nibh diam, venenatis vel eros in, egestas ullamcorper libero. Quisque id ante erat. Mauris fermentum ex vitae tristique volutpat.
+            
+            Suspendisse feugiat tempor sagittis. Morbi felis lorem, sagittis nec tellus sed, dignissim scelerisque turpis.</section>
+
+
+        <footer id="footer">
+            <section id="logo"><img src="img\Logo Parc Vacaresti.png" width="100" height="100" src=""/></section>
```

Comment:
Please check out slide 110 in our presentation for classes and ids naming convention